hack/composefiles/grafana.ini

```diff
@@ -0,0 +1,3 @@
+[security]
```

Comment:
Can't this config specify the default data-source(maybe even dashboards) so they don't need to be manually defined.

Reply:
It's another configuration file that does that. I couldn't figure out why it wasn't working so I removed it. I figured out I was putting the file in the wrong location and it wasn't being picked up so now the auto-provisioning is in there.